### DISCUSSION TITLE
Version number cosmetic

### DISCUSSION THREAD

**Discussion by hannesa2** [gitx/gitx#386] [Polls]
Version number for gitx is since years a `0.x` 
Now we publish with homebrew a signed app and what's about to move to `1.x` ?
I know it's not important, but I guess there are some people out there not using a `0.x` software.

Are there any opinions about this ?

**Comment by hannesa2** [comment]
@gitx/core

**Comment by carldunham** [comment]
Looking forward to the next release!

**Comment by hannesa2** [comment]
I released the first `1.x` version https://github.com/gitx/gitx/releases/tag/1.0
